Smewkie/pokeroguebeta
# Change Ferroseed's Hidden Ability from Iron Barbs to Anticipation

## What are the changes the user will see?
This changes Ferroseed's custom HA from Iron Barbs to Anticipation.

## Why am I making these changes?
Before, both of Ferroseed's abilities were Iron Barbs, which is confusing to the average player and caused many bug reports.
This PR aims to reduce those bug reports to 0 by just giving Ferroseed the same HA as Ferrothorn.

## What are the changes from a developer perspective?
a text change from Iron_Barbs to Anticipation.

## How to test the changes?
Spawn a Ferroseed with it's Hidden Ability, it should be Anticipation.
Or go to your starter select menu and change Ferroseed's ability to the yellow text one, it should be Anticipation now.

## Checklist
